tests/src/Unit/SocialApiExceptionTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Drupal\Tests\UnitTestCase;
+use Drupal\social_api\SocialApiException;
+
+/**
+ * Defines class for SocialApiExcetion Test.
+ */
+class SocialApiExceptionTest extends UnitTestCase {
+
+  /**
+   * Tests for class SocialApiException.
+   */
+  public function testException() {
+    $socialApiException = new SocialApiException();
```

Comment:
This tests does not make sense. If you want to test this class, just test that the message is set correctly, etc.